sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -81,7 +89,12 @@ first_session_ping_min_seq AS (
         fenix.first_session AS fenix_first_session
       WHERE
         ping_info.seq IS NOT NULL
-        AND DATE(submission_timestamp) = @submission_date
+        AND
+        {% if is_init() %}
+          DATE(submission_timestamp) >= "2019-06-21"
```

Comment:
This date doesn't match the others

Reply:
@fbertsch I'm aware, this appears to be the case in the original `init.sql` so I kept it like that. But I'm also happy to just set all these dates to match.

Reply:
I updated the date to be fixed to `"2019-06-01"`. I hope that works.

Reply:
Aug. 1, 2020 was when we started the migration to Fenix, so I would consider listing that as the start date: https://docs.telemetry.mozilla.org/concepts/analysis_gotchas.html#notable-historic-events